index.html

```diff
@@ -3226,6 +3241,21 @@ <h3>Selective Disclosure</h3>
 
       </section>
 
+      <section>
+        <h3>Other Privacy Considerations</h3>
+
+        <p>
+Since the technology described by this specification to secure documents is
+generalized in nature, the privacy implications of its use might not be
+immediately apparent to readers. In order to understand the sort of privacy
+concerns one might need to consider in a complete software system, implementers
+are urged to read about how this technology is utilized in the
+<a>verifiable credentials</a> ecosystem [[?VC-DATA-MODEL-2.0]]; see the section
+on <a data-cite="VC-DATA-MODEL-2.0#privacy-considerations">
+Verifiable Credential Privacy Considerations</a> for more information.
```

Comment:
```suggestion
Since the technology to secure documents described by this specification is
generalized in nature, the privacy implications of its use might not be
immediately apparent to readers. To understand the sort of privacy
concerns one might need to consider in a complete software system, implementers
are urged to read about how this technology is used in the
<a>verifiable credentials</a> ecosystem [[?VC-DATA-MODEL-2.0]]; see the section
on <a data-cite="VC-DATA-MODEL-2.0#privacy-considerations">
Verifiable Credential Privacy Considerations</a> for more information.
```